inlet/routing/root.go

```diff
@@ -71,6 +74,12 @@ type stopper interface {
 // Lookup uses the selected provider to get an answer. It does not return an
 // error, even when the context times out. Instead, it should just returns an
 // empty answer.
-func (c *Component) Lookup(ctx context.Context, ip netip.Addr, nh netip.Addr) provider.LookupResult {
-	return c.provider.Lookup(ctx, ip, nh)
+func (c *Component) Lookup(ctx context.Context, ip netip.Addr, nh netip.Addr, agent netip.Addr) provider.LookupResult {
+	c.metrics.routingLookups.WithLabelValues().Inc()
+	result, err := c.provider.Lookup(ctx, ip, nh, agent)
+	if err != nil {
+		c.metrics.routingLookupsFailed.WithLabelValues().Inc()
+		c.r.Warn().Err(err).Msgf("routing: error while looking up %s at %s", ip.String(), agent.String())
```

Comment:
Only log rare occurrences or use a rate limiter (check `errLoger` in `inlet/metadata/provider/snmp/root.go`)

Reply:
> Only log rare occurrences or use a rate limiter (check `errLoger` in `inlet/metadata/provider/snmp/root.go`)

Man thx for akvorado! But now i get message 404 page not found. After remove demo.yaml. what should I do?

Reply:
Implemented with a rate limiter